lib/src/status.rs

```diff
@@ -359,10 +371,11 @@ fn write_row_name(mut out: impl Write, s: &str, prefix_len: usize) -> Result<()>
 }
 
 /// Write the data for a container image based status.
-fn human_render_imagestatus(
+fn human_render_image_deployment(
```

Comment:
I feel this function may need a larger rework than just this rename.

```
● Booted image: ghcr.io/rsturla/eternal-linux/lumina:42-nvidia-open
        Digest: sha256:b1874fbb5f442657f3cc07a547c54292ca3f24fe828ab4f3132df4c9f23f1e3e (amd64)
       Version: 250425 (2025-04-25T01:46:32Z)

  Rollback image: ghcr.io/rsturla/eternal-linux/lumina:42-nvidia-open
          Digest: sha256:b1874fbb5f442657f3cc07a547c54292ca3f24fe828ab4f3132df4c9f23f1e3e (amd64)
         Version: 250425 (2025-04-25T01:46:32Z)
```

Rather than each block above being an "image", I feel it makes more sense being a "deployment" since it contains more information than just the deployment.  For example, the index, pinned state, /usr overlay state etc.

Reply:
I've finally got round to implementing these changes.

Have rebased onto the current main branch, and updated based on the other comments.